src/pages/partners/affiliate-ib/_why-us.tsx

```diff
@@ -24,11 +24,6 @@ type RowProps = {
 }
 const query = graphql`
     query {
-        multiple_income_opportunities: file(
-            relativePath: { eq: "affiliate/multiple-income-opportunities.png" }
```

Comment:
Check this image is being used anywhere else. If not then delete the image as well.

Reply:
The image is not being used in other files. Already remove it.